tests/test_idealize.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+import biotite.structure as struc
+from peppr.idealize import idealize_bonds
+
+
+def make_corrupted_pose() -> struc.AtomArray:
```

Comment:
As `make_corrupted_pose()` and `make_clashing_pose()` do not require parameters, I think it is more idiomatic to use a fixture for them that you can request in the parameters of the test functions
```suggestion
@pytest.fixture
def corrupted_pose() -> struc.AtomArray:
```

Reply:
Of course. Good call.